book-my-show/backend/java/bms-monolith/src/main/java/org/lbcc/bms/bms_monolith/entity/Seat.java

```diff
@@ -0,0 +1,34 @@
+package org.lbcc.bms.bms_monolith.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.EnumType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.lbcc.bms.bms_monolith.common.entity.BaseAuditingEntity;
+import org.lbcc.bms.bms_monolith.entity.enums.OperationalStatus;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Seat extends BaseAuditingEntity {
+
+    @ManyToOne
+    @JoinColumn(name = "seat_type_id")
+    private SeatType seatType;
+
+    @ManyToOne
+    @JoinColumn(name = "venue_id")
+    private Venue venue;
+
+    @Enumerated(EnumType.STRING)
+    private OperationalStatus operationalStatus;
+
```

Comment:
@SiZe(min = 4, max = 15, message = "Label should be between 4 and 15 characters")
@column(length = 15, nullable = false)
private String label;